### PR TITLE
added 3 of my mods

### PR DESCRIPTION
**Pull request by art0007i**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3505,6 +3505,75 @@
...
+                        {
+                            "url": "https://github.com/art0007i/CapturePrivateUI/releases/download/1.0.0.0/CapturePrivateUI.dll",
+                            "filename": "CapturePrivateUI.dll",
+                            "sha256": "a3a66d961556e8c7a3386fd34ade1e4dbe4b191ad76af6103255f3fc186efc68"
+                        }
+                    ]
+                }
+            }
+	},
+        "me.art0007i.CustonInspectors": {
```
Comment (ljoonal) [inline manifest.json:3532]: Custon vs Custom ?

**Review by ljoonal — APPROVED** [review]

File: manifest.json [hunk 2]
```diff
@@ -3505,6 +3505,75 @@
...
+            "name": "CustonInspectors",
+            "description": "Lets you change your default inspector panel",
+            "category": "Inspectors",
+            "sourceLocation": "https://github.com/art0007i/CustonInspectors",
```
Comment (XDelta) [inline manifest.json:3536]: 404 link, Custon instead of custom

**Comment by EIA485** [comment]
resolved merge conflicts

**Review by ljoonal — CHANGES_REQUESTED** [review]
404 links Delta pointed out should be fixed.

File: manifest.json (hunk 2 above)
Comment (EIA485) [reply manifest.json:3555]: name is also set too "CustonInspectors"

**Comment by art0007i** [comment]
ok should be good now

**Review by art0007i — COMMENTED** [review]
.
what does this mean, I reviewed my own thing?
it still says changes requested even though I have made a change? idk how github works

**Review by ljoonal — APPROVED** [review]

**Pull request merged**